src/frontend/Sider/Behandling/Totrinnskontroll/typer.ts

```diff
@@ -15,12 +15,17 @@ export type TotrinnskontrollResponse =
           totrinnskontroll: TotrinnskontrollUnderkjentResponse;
       }
     | {
-          status: TotrinnskontrollStatus.KAN_FATTE_VEDTAK | TotrinnskontrollStatus.UAKTUELT;
+          status: TotrinnskontrollStatus.KAN_FATTE_VEDTAK;
+          totrinnskontroll: TotrinnskontrollOpprettet;
+      }
+    | {
+          status: TotrinnskontrollStatus.UAKTUELT;
       };
 
 export type TotrinnskontrollOpprettet = {
     opprettetAv: string;
     opprettetTid: string;
+    begrunnelse?: string;
```

Comment:
Er "begrunnelse" og "kommentarTilBeslutter" det samme? Hadde nok i så fall foretrukket at de ikke het to forskjellige ting.  Spesielt siden "begrunnelse" brukes mye i sammenheng med vilkårene.

Reply:
Begge lagres ned på feltet "begrunnelse" i databasen fordi dette feltet allerede fantes. Alle dto-er forholder seg til ulike navn fordi det skal brukes ulike steder. Vises ulike steder og brukes på ulike måter. Kanskje ikke optimalt, men syns også det hadde vært litt forvirrende om de het det samme, spesielt i historikken